### PR TITLE
BIT-273 NEO: Implement claim transactions executor

### PR DESCRIPTION
**Pull request by some-dev**

### CODE REVIEW

File: src/Lykke.Job.NeoClaimTransactionsExecutor.AzureRepositories/AggregateKeysBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    internal static class AggregateKeysBuilder
+    {
+        public static string BuildPartitionKey(Guid aggregateId)
+        {
+            return aggregateId.ToString();
+        }
+
+        public static string BuildRowKey()
+        {
+            return "_";
```
Comment (cassade) [inline src/Lykke.Job.NeoClaimTransactionsExecutor.AzureRepositories/AggregateKeysBuilder.cs]: String.Empty?

**Review by cassade — DISMISSED** [review]

File: src/Lykke.Job.NeoClaimTransactionsExecutor.AzureRepositories/TransactionExecution/TransactionExecutionEntity.cs [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    {
+        public Guid TransactionId { get; set; }
+
+        public string Address { get; set; }
+
+        public string NeoAssetId { get; set; }
+
+        public string GasAssetId { get; set; }
+
+        public string NeoBlockchainIntegrationLayerId { get; set; }
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.NeoClaimTransactionsExecutor.AzureRepositories/TransactionExecution/TransactionExecutionEntity.cs]: Let's rename it to NeoBlockchainType here and in the everywhere in the project

File: src/Lykke.Job.NeoClaimTransactionsExecutor.AzureRepositories/TransactionExecution/TransactionExecutionEntity.cs [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+        public string NeoBlockchainIntegrationLayerAssetId { get; set; }
+
+        public string GasBlockchainIntegrationLayerId { get;  set; }
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.NeoClaimTransactionsExecutor.AzureRepositories/TransactionExecution/TransactionExecutionEntity.cs]: Let's rename it GasBlockchainType here and in the everywhere in the project

File: src/Lykke.Job.NeoClaimTransactionsExecutor.AzureRepositories/TransactionExecution/TransactionExecutionEntity.cs [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+        public string Address { get; set; }
+
+        public string NeoAssetId { get; set; }
+
+        public string GasAssetId { get; set; }
+
+        public string NeoBlockchainIntegrationLayerId { get; set; }
+
+        public string NeoBlockchainIntegrationLayerAssetId { get; set; }
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.NeoClaimTransactionsExecutor.AzureRepositories/TransactionExecution/TransactionExecutionEntity.cs]: Let's rename it NeoBlockchainAssetId here and in the everywhere in the project

File: src/Lykke.Job.NeoClaimTransactionsExecutor.AzureRepositories/TransactionExecution/TransactionExecutionEntity.cs [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+        public string GasBlockchainIntegrationLayerId { get;  set; }
+
+        public string GasBlockchainIntegrationLayerAssetId { get; set; }
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.NeoClaimTransactionsExecutor.AzureRepositories/TransactionExecution/TransactionExecutionEntity.cs]: Let's rename it GasBlockchainAssetId here and in the everywhere in the project

File: src/Lykke.Job.NeoClaimTransactionsExecutor/Startup.cs [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+                options.Logs = logs =>
+                {
+                    logs.AzureTableName = "NeoClaimTransactionsExecutorJobLog";
+                    logs.AzureTableConnectionStringResolver = settings => settings.NeoClaimTransactionsExecutorJob.Db.LogsConnString;
+
+                    // TODO: You could add extended logging configuration here:
+
+                    logs.Extended = p =>
+                    {
+                        p.AddAdditionalSlackChannel("CommonBlockChainIntegration");
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.NeoClaimTransactionsExecutor/Startup.cs]: Add these lines for both additional channels https://github.com/LykkeCity/Lykke.Job.BlockchainCashoutProcessor/blob/master/src/Lykke.Job.BlockchainCashoutProcessor/Startup.cs#L77-L78

File: settings.yaml [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    GasAssetId:
+        settings-key: NeoClaimTransactionsExecutorJob-GasAssetId
+    SignFacadeApiKey:
+        settings-key: NeoClaimTransactionsExecutorJob-SignFacadeApiKey
+    SignFacadeUrl:
+        settings-key: NeoClaimTransactionsExecutorJob-SignFacadeUrl
+    NeoApiUrl:
+        settings-key: NeoClaimTransactionsExecutorJob-NeoApiUrl
+    AssetsServiceUrl:
+        settings-key: NeoClaimTransactionsExecutorJob-AssetsServiceUrl
```
Comment (KonstantinRyazantsev) [inline settings.yaml]: -> AssetsServiceUrl

File: settings.yaml [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+    NeoAssetId:
+        settings-key: NeoClaimTransactionsExecutorJob-NeoAssetId
+    GasAssetId:
+        settings-key: NeoClaimTransactionsExecutorJob-GasAssetId
+    SignFacadeApiKey:
+        settings-key: NeoClaimTransactionsExecutorJob-SignFacadeApiKey
+    SignFacadeUrl:
+        settings-key: NeoClaimTransactionsExecutorJob-SignFacadeUrl
+    NeoApiUrl:
+        settings-key: NeoClaimTransactionsExecutorJob-NeoApiUrl
```
Comment (KonstantinRyazantsev) [inline settings.yaml]: -> BilNeoApiUrl

File: settings.yaml [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    NeoHotWalletAddress:
+        settings-key: NeoClaimTransactionsExecutorJob-NeoHotWalletAddress
+    NeoAssetId:
+        settings-key: NeoClaimTransactionsExecutorJob-NeoAssetId
+    GasAssetId:
+        settings-key: NeoClaimTransactionsExecutorJob-GasAssetId
+    SignFacadeApiKey:
+        settings-key: NeoClaimTransactionsExecutorJob-SignFacadeApiKey
+    SignFacadeUrl:
+        settings-key: NeoClaimTransactionsExecutorJob-SignFacadeUrl
```
Comment (KonstantinRyazantsev) [inline settings.yaml]: BilSignFacadeUrl

File: settings.yaml [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+        RabbitConnectionString:
+            settings-key: CqrsRabbitConnString
+            types:
+                - RabbitMq
+        RetryDelay:        
+            settings-key: NeoClaimTransactionsExecutorJob-Cqrs-RetryDelay
+        WaitForTransactionRetryDelay:        
+            settings-key: NeoClaimTransactionsExecutorJob-Cqrs-WaitForTransactionRetryDelay            
+    NeoHotWalletAddress:
+        settings-key: NeoClaimTransactionsExecutorJob-NeoHotWalletAddress
```
Comment (KonstantinRyazantsev) [inline settings.yaml]: -> BilNeoHotWalletAddress

File: settings.yaml [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+    NeoAssetId:
+        settings-key: NeoClaimTransactionsExecutorJob-NeoAssetId
```
Comment (KonstantinRyazantsev) [inline settings.yaml]: -> NeoAssetId

File: settings.yaml [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+    GasAssetId:
+        settings-key: NeoClaimTransactionsExecutorJob-GasAssetId
```
Comment (KonstantinRyazantsev) [inline settings.yaml]: -> NeoGasAssetId

**Review by KonstantinRyazantsev — APPROVED** [review]

**Pull request merged**